resources/js/app.js

```diff
@@ -19,8 +19,15 @@
    otherwise({
     redirectTo: '/home'
    });
+       $locationProvider.html5Mode({
```

Comment:
Yes 👍  lets add the indentation bear, it will be really useful.

@aalind0 👍  😄  It's alright! its the whole point of review right 😉  once we have indentation bear here, it'll be a really straightforward process to get rid of indentation problems. 😃